rocketpy/rocket/rocket.py

```diff
@@ -582,7 +606,7 @@ def evaluate_inertias(self):
         """
         # Get masses
         prop_mass = self.motor.propellant_mass  # Propellant mass as a function of time
-        dry_mass = self.dry_mass  # Constant rocket dry mass without propellant
+        dry_mass = self.dry_mass  # Constant rocket mass without motor
```

Comment:
```suggestion
        dry_mass = self.dry_mass  # Constant rocket mass with motor, without propellant
```

This includes motor mass